python/gym_jiminy/common/setup.py

```diff
@@ -62,7 +62,8 @@
         # - 0.18.0: dtype handling of flatten space
         # - >=0.18.0,<0.18.3 requires Pillow<8.0 to work, not compatible with
         #   Python 3.9.
-        "gym>=0.18.3"
+        "gym>=0.18.3,<0.21.0"
```

Comment:
`gym>=0.18.3,<=0.21.0` is actually supported.

Reply:
By the way, I think it is only [gym-jiminy-rllib](https://github.com/duburcqa/jiminy/blob/3432d600e88d921378f6c34af131d1f7ac0a7dcf/python/gym_jiminy/rllib/setup.py#L32) that actually requires `<=0.21.0` (instead of `<=0.24.0` for `gym_jiminy-common` on dev branch). I have not double checked though.